requirements.txt

```diff
@@ -3,6 +3,7 @@ wagtail==5.0.5
 wagtailmedia
 psycopg2-binary
 wagtail-content-import
+wagtail-localize
```

Comment:
Taking a brief look at the docs, this gives us integrations with external translation services, and the ability to work with PO files, which are not things that we've really needed for contentrepo. What does this give us that we want to include it?

Reply:
This allows the user to create a `HomePage` with a non default locale as far as I understand. Without it, the admin UI will still allow you to create locales, but the `Homepage` with that locale will not be created(which is something we want).

Reply:
On my local instance (without these changes), if I go settings -> Locales, and then add a locale. I can then either go to the english homepage, click the kebab menu and select translate, which will copy that whole tree into the new locale, creating the new homepage with all the required pages.

Reply:
I didn't know that. That works for me too. I will remove it then.